src/components/AnonymousChat/PostNotificationChannelItem.tsx

```diff
@@ -87,7 +89,7 @@ const PostNotificationChannelItem: (props: MessageChannelItemProps) => React.Rea
       comments={item?.rawJson?.comments?.length}
       downvote={item?.rawJson?.downvote}
       isCommentExists={Boolean(firstComment)}
-      isMe={item?.user?.isMe || isOwnSignedPost || isOwnPost}
+      isMe={getIsMe()}
       isOwnSignedPost={isOwnSignedPost}
       message={item?.description}
       name={postMaker?.data?.username}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [92-97]

The new code changes the condition for setting `isMe`. It now calls the `getIsMe` function instead of checking `item?.user?.isMe`, `isOwnSignedPost`, or `isOwnPost`. This could potentially change the behavior of the application if the `getIsMe` function does not cover all cases previously handled by the old condition. Please ensure that this change does not affect other parts of the application.
```diff
-  isMe={item?.user?.isMe || isOwnSignedPost || isOwnPost}
+  isMe={getIsMe()}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->